packages/salesforcedx-vscode-core/package.json

```diff
@@ -92,6 +92,7 @@
         "@salesforce/source-tracking": "4.2.14",
         "@salesforce/templates": "^58.0.1",
         "@salesforce/source-deploy-retrieve": "9.7.18",
+        "jsforce": "1.11.0",
```

Comment:
Note this is in the packageUpdates property.  These are deps that are directly installed b/c they can't be bundled.  In this case we are ensuring that the version of sf-core in the @heroku/functions/core dependancy finds the right version of jsforce.